hawc/apps/lit/forms.py

```diff
@@ -548,3 +549,50 @@ def clean_excel_file(self):
                 '"Full text URL", case sensitive.'
             )
         return fn
+
+
+class BulkReferenceStudyExtractForm(forms.Form):
+    reference_ids = forms.ModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple, queryset=models.Reference.objects.none(), required=True
+    )
+    study_type = forms.TypedMultipleChoiceField(
```

Comment:
do we need to add `required=True` here too?

Reply:
I was under the impression someone may wish to create a study without a study type, but if that's not true we should change it

Reply:
Ahh, that makes sense; they could just want to perform study evaluation. Let's leave as-is, good thinking.